components/ui/header.tsx

```diff
@@ -5,41 +5,99 @@ import PearLightLogo from "./PearLight70x70.svg";
 
 export default function Header() {
   return (
```

Comment:
before we return the function/component we can add a navLinks array to simplify our code later
```js
  const navLinks = [
    { label: "About", path: "/about" },
    { label: "Discord", path: "https://discord.gg/AKy5FmqCkF" },
    { label: "Github", path: "https://github.com/trypear/pearai-app" },
  ];
```

Reply:
done